java/build.gradle

```diff
@@ -22,6 +22,10 @@ dependencies {
     implementation 'com.google.code.gson:gson:2.11.0'
     implementation 'org.jetbrains:annotations:26.0.1'
     implementation 'io.github.cdimascio:dotenv-java:3.1.0'
+    implementation 'com.ibm.async:asyncutil:0.1.0'
+    implementation 'com.amazonaws:aws-java-sdk-s3:1.12.780'
```

Comment:
This should probably be a `testImplementation` right? We don't use S3 outside of the new test?